samples/httpbin/sample-client/fortio-deploy.yaml

```diff
@@ -15,6 +15,8 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: fortio-deploy
+  labels:
+    app: fortio
```

Comment:
Slight mismatch on name and app.

Reply:
Removed based on https://github.com/istio/istio/pull/27242#issuecomment-692549416